_data/simple-icons.json

```
@@ -3442,6 +3442,11 @@
             "hex": "79BD42",
             "source": "https://comicfury.com/images/gator-icon-black.png"
         },
+        {
+            "title": "comma",
+            "hex": "51FF00",
+            "source": "https://comma.ai/images/logo-icon-shape.svg"
```

```suggestion
            "source": "https://comma.ai"
```
Given the SVG is in the source code of the homepage, let's tidy this up.